infrastructure/prismic-snapshots/README.md

```diff
@@ -0,0 +1,216 @@
+
+## Purpose
+
+Maintains 14 days of rolling snapshots for all Prismic document content. Snapshots are generated at 11 PM UTC.
+
+## File Structure
+
+```
+infrastructure/prismic-snapshots/
+├── terraform.tf              # Provider configuration
+├── main.tf                   # Main infrastructure
+├── outputs.tf               # Terraform outputs
+├── README.md                # This file
+├── build-lambda.sh          # Script to build Lambda package with dependencies
+├── deploy.sh                # Full infrastructure deployment
+├── deploy-code.sh           # Code-only deployment script
+├── lambda/
+│   └── prismic_snapshot.js   # Lambda function code
+└── prismic_snapshot_lambda.zip # Generated deployment package
+```
+
+## Architecture
+
+```
+┌─────────────────┐    ┌──────────────────┐    ┌─────────────────┐
+│ EventBridge     │───▶│ Lambda Function  │───▶│ S3 Bucket       │
+│ (Daily 11 PM)   │    │ (Download)       │    │ (Store + Clean) │
+└─────────────────┘    └──────────────────┘    └─────────────────┘
+                              │
+                              ▼
+                       ┌──────────────────┐
+                       │ CloudWatch Logs  │
+                       │ (Monitoring)     │
+                       └──────────────────┘
+```
+
+## Configuration
+
+### Schedule
+
+The Lambda runs daily at **11 PM UTC**. To change this, modify the `schedule_expression` in `main.tf`:
+
+```hcl
+schedule_expression = "cron(0 23 * * ? *)" # 11 PM UTC daily
+```
+
+### Retention
+
+Snapshots are kept for **14 days**. To change this, modify the `days` in the lifecycle configuration in `main.tf`:
+
+```hcl
+expiration {
+  days = 14  # Change this value
+}
+```
+
+### Memory/Timeout
+
+Lambda is configured with:
+
+- **1GB memory**
+- **15 minute timeout**
+
+Adjust in `main.tf` if needed based on Prismic content size.
+
+## Snapshot Storage
+
+### S3 Bucket Location
+
+**Bucket**: `wellcomecollection-prismic-backups`
+
+### Filename Structure
+
+Snapshots are stored with ISO 8601 timestamp filenames:
+
+```
+prismic-snapshot-YYYY-MM-DDTHH-MM-SSZ.json
+```
+
+**Examples**:
+
+- `prismic-snapshot-2025-11-03T23-00-15Z.json` (daily backup at 11 PM UTC)
+- `prismic-snapshot-2025-11-04T23-00-22Z.json` (next day's backup)
+
+### File Content
+
+Each snapshot contains the complete prismic repository content data for wellcomecollection.org in JSON format
+
+## Deployment
+
+### Prerequisites
+
+1. **AWS CLI configured** with appropriate permissions
+2. **Terraform installed** (>= 0.12)
+3. **Node.js and npm installed** (for building Lambda package)
+4. **Prismic access token** stored in AWS Secrets Manager
+
+N.B. The Prismic access token is already stored in AWS Secrets Manager as `prismic-model/prod/access-token` and is used by other services. You can verify it exists:
+
+```bash
+AWS_PROFILE=experience-developer aws secretsmanager describe-secret --secret-id "prismic-model/prod/access-token"
```

Comment:
does this need updating too or are we eventually going to try and use the secret from the experience account?